Makefile

```
@@ -1654,6 +1654,7 @@ config.status:
 	@echo "- In file README.win32.adoc for Windows systems."
 	@echo "On Unix systems, if you've just unpacked the distribution,"
 	@echo "something like"
+	@echo "  tools/autogen (needed only for git clones)"
```

This whole thing has been broken since 4.13 (see #11347!), so I'm hesitant suggesting, but this could be:
```suggestion
ifeq "$(wildcard configure)" ""
	@echo "  tools/autogen"
endif
```
and only suggest the command if it's clearly needed. This would stop the command being suggested if you extracted a release tarball and just ran `make`, for example.